test/functionalTest/cases/117_convop_using_standard_ops/getIndividualContextEntity.test

```diff
@@ -0,0 +1,551 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+ConvOp getIndividualContextEntity: GET /v1/contextEntities/{entityId::id}
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB 0
+
+--SHELL--
+#
+# FIXME P5: Once the broker implements forward queries for empty entity types, this test case should
+#           include forwarding.
+#
+# FIXME P5: This convenience operation can only respond with ONE entity.
+#           So, to avoid problems, this functest is designed to have only one matching entity
+#           for each query.
+#           Once/If the broker changes this behaviour and permits the response to this conv op
+#           to include a vector of context elements, this func test should be enhanced to include
+#           tests of this.
+#
+# 01. Update/APPEND E1/T1/A1 without service-path
+# 02. Update/APPEND E1//A1 with service-path /A1
+# 03. Update/APPEND E1/T1/A2 with service-path /A2
+# 04. Update/APPEND E1//A2 with service-path /A3
+# 05. Update/APPEND E2/T2/A2 with service-path /A2
+# 06. Update/APPEND E3//A2 with service-path /A2
+# 07. Query E2 with service-path '/A2'
+# 08. Query E1 with service-path '/A2' and URI param !exist=entity::type
+# 09. Query E1 with service-path '/A2' and URI param exist=entity::type
+# 10. Query E1 without service-path and URI param entity::type=T1
+# 11. Query E3 with service-path '/A2' and URI param !exist=entity::type
+#
```

Comment:
This operation should support `?attributesFormat` so I'd suggest to include some steps using it.